public/windows.html

```diff
@@ -319,8 +319,8 @@ <h2 class="step-title">2</h2>
                 class="svg-fill"
               >
                 %USERPROFILE%\Pictures\Screenshots
-              </text> -->
-            </svg>
+              </text>
+            </svg> -->
```

Comment:
Good catch, thanks! 😄

Reply:
no problem! just happened to notice it while running the prettier script.